datalad/__init__.py

```diff
@@ -16,6 +16,11 @@
 from .config import ConfigManager
 cfg = ConfigManager()
 
+from datalad.support.sshconnector import SSHManager
```

Comment:
doh -- I have merged it without mentioning that this comment wasn't addressed... ok -- I will send a quick PR